Description: googletest and gmock are packaged, use those
 No need to add external (empty) project directories, will link with system
 ones357.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: not-needed
Last-Update: 2024-05-01

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -330,20 +330,20 @@
   set(build_gmock ON)
 
   # This project is tested using GoogleTest.
-  add_subdirectory("third_party/googletest")
+  # add_subdirectory("third_party/googletest")
 
   # GoogleTest triggers a missing field initializers warning.
-  if(SNAPPY_HAVE_NO_MISSING_FIELD_INITIALIZERS)
-    set_property(TARGET gtest
-        APPEND PROPERTY COMPILE_OPTIONS -Wno-missing-field-initializers)
-    set_property(TARGET gmock
-        APPEND PROPERTY COMPILE_OPTIONS -Wno-missing-field-initializers)
-  endif(SNAPPY_HAVE_NO_MISSING_FIELD_INITIALIZERS)
-
-  if(SNAPPY_HAVE_NO_IMPLICIT_INT_FLOAT_CONVERSION)
-    set_property(TARGET gtest
-        APPEND PROPERTY COMPILE_OPTIONS -Wno-implicit-int-float-conversion)
-  endif(SNAPPY_HAVE_NO_IMPLICIT_INT_FLOAT_CONVERSION)
+  # if(SNAPPY_HAVE_NO_MISSING_FIELD_INITIALIZERS)
+  #   set_property(TARGET gtest
+  #       APPEND PROPERTY COMPILE_OPTIONS -Wno-missing-field-initializers)
+  #   set_property(TARGET gmock
+  #       APPEND PROPERTY COMPILE_OPTIONS -Wno-missing-field-initializers)
+  # endif(SNAPPY_HAVE_NO_MISSING_FIELD_INITIALIZERS)
+  # 
+  # if(SNAPPY_HAVE_NO_IMPLICIT_INT_FLOAT_CONVERSION)
+  #   set_property(TARGET gtest
+  #       APPEND PROPERTY COMPILE_OPTIONS -Wno-implicit-int-float-conversion)
+  # endif(SNAPPY_HAVE_NO_IMPLICIT_INT_FLOAT_CONVERSION)
 
   add_executable(snappy_unittest "")
   target_sources(snappy_unittest
@@ -376,7 +376,7 @@
   # This project uses Google benchmark for benchmarking.
   set(BENCHMARK_ENABLE_TESTING OFF CACHE BOOL "" FORCE)
   set(BENCHMARK_ENABLE_EXCEPTIONS OFF CACHE BOOL "" FORCE)
-  add_subdirectory("third_party/benchmark")
+  # add_subdirectory("third_party/benchmark")
 endif(SNAPPY_BUILD_BENCHMARKS)
 
 if(SNAPPY_FUZZING_BUILD)
